test/lib/controller/init.js

```diff
@@ -16,9 +16,14 @@ chai.should();
 describe('For init controller', () => {
```

Comment:
I don't understand the rest of the code in this module. We will have to meet and discuss the same.